### DISCUSSION TITLE
Upgrade to version 0.8.1

### DISCUSSION THREAD

**Discussion by ppfromero** [mjhoptics/ray-optics#87] [Q&A] [answered]
Hello,

I want to use flip function included in 0.8.1 version.
My current version is 0.7.5
There is a simple way to upgrade?

I try 

conda upgrade -c conda-forge rayoptics
I get message : "All requested packagesalready installed"

thanks for your help

**Comment by mjhoptics** [comment, marked as answer]
Try:

`conda update -c conda-forge rayoptics`

You can also search to see what packages are available:

`conda search -c conda-forge rayoptics`

Mike

**Reply by ppfromero** [reply]
Thanks Mike I update all my env. Because dependecies need to also to be updated.
Now I use 0.8.1

BR
José
